backend/test/resolver.spec.js

```diff
@@ -0,0 +1,63 @@
+const {ApolloServer, gql} = require('apollo-server-express');
+const {createTestClient} = require('apollo-server-testing');
+const typeDefs = require('../schema.js');
+const resolvers = require('../resolvers.js');
+
+const server = new ApolloServer({
+    typeDefs,
+    resolvers
+});
+
+const {query, mutate} = createTestClient(server);
+
+// could not outsource these queries/mutations...
+const ALL_ITEMS_QUERY = gql`
+    query itemsQuery {
+        items {
+            id,
+            text,
+            editMode
+        }
+    }
+`;
+
+const CREATE_ENTRY = gql`
+    mutation createEntry($text: String!) {
+        createEntry(text: $text) {
+            id,
+            text,
+            editMode
+        }
+    }
+`;
+
+const DELETE_ENTRY = gql`
+    mutation DeleteEntry($id: String!) {
+        deleteEntry(id: $id)
+    }
+`;
+
+// todo's: add beforeEach's, add more/better test cases
+describe('Queries', () => {
+    it("has start todo items", () => {
+        query({query: ALL_ITEMS_QUERY}).then((data) => {
+            expect(data.data.items).toHaveLength(4);
```

Comment:
Nur die Datenlänge ist vielleicht etwas wenig als Test. Besser wäre hier eine zusätzliche Prüfung des Inhalts, z.B. so.
```suggestion
            expect(data.data.items).toHaveLength(4);
            expect(data.data.items).toMatchObject([
                {id: 1},
                {id: 2},
                {id: 3},
                {id: 4}
                ]);
```

Reply:
nicer tip! Hab ich implementiert!